components/navigation/navigation.css

```diff
@@ -44,3 +44,61 @@
 .nav-fixed-adjust {
   margin-top: 5rem;
 }
+
+.nav a {
+  color: inherit;
+  text-decoration: none;
+}
+
+/* Drawer */
+
+.drawer-frame {
+  display: flex;
+  overflow: hidden;
+  height: 100vh;
+}
+
+.drawer {
+  display: flex;
+  flex-direction: column;
+  flex: 0 0 16rem;
+  border-right: 1px solid var(--light-grey);
+  height: 100%;
+}
+
+.drawer-header {
+  padding: 1rem 1rem;
+}
```

Comment:
Suggestion: You can use shorthand for `padding`

Reply:
Thanks for the suggestion, will fix that.